library/crates/Cargo.toml

```diff
@@ -31,6 +35,7 @@ ring = "=0.16.20"
 rocksdb = "=0.17.0"
 rustix = "=0.35.9"
 serde = "=1.0.145"
+serde_json = "1.0"
```

Comment:
Add a JSON library as TypeDB Cloud needs to communicate with Auth0 using JSON data structure.